google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -407,6 +416,40 @@ def storage_class= new_storage_class
           patch_gapi! :storage_class
         end
 
+        ##
+        # Whether Autoclass is enabled for the bucket.
+        #
+        # @return [Boolean]
+        #
+        def autoclass_enabled
+          @gapi.autoclass&.enabled?
+        end
+
+        ##
+        # Toggle time of the autoclass
+        #
+        # @return [DateTime]
+        #
+        def autoclass_toggle_time
+          @gapi.autoclass&.toggle_time
+        end
+
+        ##
+        # Updates bucket's autoclass configuration. This defines the default class for objects in the
+        # bucket and down/up-grades the storage class of objects based on the access patterns.
+        # Accepted values are `:false`, and `:true`.
+        # rubocop:todo update the link
```

Comment:
What does this line do?

Reply:
Todo reference for future.

Reply:
Just confirmed the docs link. Feel free to add it in. Thanks for your patience!

https://cloud.google.com/storage/docs/using-autoclass